test-files/old-tests/param-direction.cpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * @param x0
```

Comment:
When there are tests for javadocs, they should be strictly javadoc tests (I think yours are) and they should have a parent directory in the chain which is strictly javadoc tests